_includes/footer.html

```diff
@@ -1,2 +1,43 @@
 <footer>
+  <div class="bg-blue-50/50">
+    <div class="pt-16 pb-12 px-8 container mx-auto">
+      <div class="flex justify-center max-w-5xl mx-auto">
+        <div class="w-1/3">
+          <a href="/">
+            <img src="{{ site.baseurl }}/assets/images/logo-dark.png" alt="Growht Nirvana" class="h-10">
+          </a>
+          <p class="mt-3 leading-7">
+            Growth Nirvana Inc.
+            <br>
+            145 Cedar Way #14
+            <br>
+            Laguna Beach, CA
+            <br>
+            92651
+          </p>
+        </div>
+        <div class="w-1/3 flex flex-col space-y-4">
+          <h3 class="font-semibold text-2xl">Resources</h3>
+          <a href="/blog" class="pl-2 font-semibold hover:text-primary">Blog</a>
+          <a href="/glossary" class="pl-2 font-semibold hover:text-primary">Glossary</a>
+          <a href="/connectors" class="pl-2 font-semibold hover:text-primary">Data Connectors</a>
+          <a href="/case-study" class="pl-2 font-semibold hover:text-primary">Case Studies</a>
+          <a href="/faqs" class="pl-2 font-semibold hover:text-primary">FAQs</a>
+        </div>
+        <div class="w-1/3 flex flex-col space-y-4">
+          <h3 class="font-semibold text-2xl">Compnay</h3>
+          <a href="/about-us" class="pl-2 font-semibold hover:text-primary">About us</a>
+          <a href="#schedule-demo" class="pl-2 font-semibold hover:text-primary">Contact us</a>
+          <a href="/msa" class="pl-2 font-semibold hover:text-primary">Terms of Service</a>
+          <a href="/privacy-policy" class="pl-2 font-semibold hover:text-primary">Privacy Policy</a>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="bg-blue-50">
+    <div class="py-5 px-8 container mx-auto flex justify-center">
+      Copyright © 2023 <a href="/" class="text-primary font-semibold mx-2">Growth Nirvana.</a> All rights reserved.
```

Comment:
Change the year to a liquid date filter:
```suggestion
      Copyright © {{ "now" | date: "%Y" }} <a href="/" class="text-primary font-semibold mx-2">Growth Nirvana.</a> All rights reserved.
```